backend/app/assets/stylesheets/spree/backend/sections/_orders.scss

```diff
@@ -49,6 +49,14 @@
   margin-bottom: 18px;
 }
 
+[data-hook="complete-order-button"] {
+  margin-bottom: 15px;
+}
+
+#billing {
+  margin-top: 45px;
```

Comment:
Every div with an id of `billing` now gets a top margin? Could we use a more specific selector instead? Or add a utility class that handles these cases? There are probably more places in admin where a form is built like this form?

Reply:
You are correct. There are also other uses of this partial where this rule needs to be applying. I'll make this selector more specific and it probably needs to live in a different SCSS file as well. Not sure what I was thinking when I added this 😬 Thanks!